beets/ui/commands.py

```diff
@@ -1479,6 +1480,13 @@ def move_items(lib, dest, query, copy, album, pretend, confirm=False):
             show_path_changes([(obj.path, obj.destination(basedir=dest))
                                for obj in objs])
     else:
+        if export:
+            if album:
+                util.copy([(item.path, item.destination(basedir=dest))
+                                   for obj in objs for item in obj.items()])
+            else:
+                util.copy([(obj.path, obj.destination(basedir=dest))
+                                   for obj in objs])
```

Comment:
Here, you're just passing one argument—a list, produced by a list comprehension—to the `copy` function. Maybe you mean something like:

```
for item in objs:
    util.copy(item.path, item.destination...)
```

and then again with `for item in obj.items()`.